src/command.rs

```diff
@@ -50,7 +50,10 @@ impl IdentifyChain for dyn sc_service::ChainSpec {
 	fn identify(&self) -> ChainIdentity {
 		if self.id().starts_with("centrifuge") {
 			ChainIdentity::Centrifuge
-		} else if self.id().starts_with("altair") || self.id().starts_with("charcoal") {
+		} else if self.id().starts_with("altair")
+			|| self.id().starts_with("charcoal")
+			|| self.id().starts_with("antares")
+		{
```

Comment:
Can we put these into development identiy?

Reply:
There will be a development environment for centrifuge soon, before we go live in March. Charcoal will most likely disappear from this list. This two envs `Antares` & `Axis/Catalyst` should use the same as their parent network. `Development` is for non-long-term environments or internal only.

Reply:
This is only used to guess the right `ChainSpec` to use for custom JSON chain specifications - see https://github.com/centrifuge/centrifuge-chain/pull/572/files#diff-de6b9cfd1a256da7900d35e26aa7279bd713110e68e91b52616befb7397848dfR90-R101